Source/ACE/Entity/Landblock.cs

```diff
@@ -321,16 +321,14 @@ private Position GetWorldObjectPosition(ObjectGuid objectId)
 
         public float GetWorldObjectEffectiveUseRadius(ObjectGuid objectId)
         {
-            WorldObject wo;
             Log($"Getting WorldObject Effective Use Radius {objectId.Full:X}");
 
-            wo = worldObjects.ContainsKey(objectId) ? worldObjects[objectId] : null;
-            if (wo != null)
-            {
-                var csetup = SetupModel.ReadFromDat(wo.SetupTableId.Value);
-                return (float)Math.Pow((wo.UseRadius.Value + csetup.Radius + 1.5), 2);
-            }
-            return 0.00f;
+            WorldObject wo = worldObjects.ContainsKey(objectId) ? worldObjects[objectId] : null;
+            if (wo?.SetupTableId == null) return 0.00f;
```

Comment:
I'm pretty sure we have a no conditional-on-same-line policy.  (Personally, I strongly believe in always using curly braces as well, but I wont enforce that)